packages/esm-home-app/src/homepage-header/homepage-header.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Location } from '@carbon/react/icons';
+import { ConfigObject, useConfig, useSession } from '@openmrs/esm-framework';
+import { DatePicker, DatePickerInput } from '@carbon/react';
+import dayjs from 'dayjs';
+import styles from './homepage-header.scss';
+import HomepageIllustration from './homepage-illustration.component';
+
+interface HomepageHeaderProps {
+  dashboardTitle: string;
+}
+
+const HomepageHeader: React.FC<HomepageHeaderProps> = ({ dashboardTitle }) => {
+  const { t } = useTranslation();
+  const session = useSession();
+  const location = session?.sessionLocation?.display;
+  const config = useConfig() as ConfigObject;
+
+  return (
+    <div className={styles.header} role="banner">
+      <div className={styles['left-justified-items']}>
+        <HomepageIllustration />
+        <div className={styles['page-labels']}>
+          <p>{config.appNameLabel}</p>
+          <p className={styles['page-name']}>{dashboardTitle}</p>
+        </div>
+      </div>
+      <div className={styles['right-justified-items']}>
+        <div className={styles['date-and-location']}>
+          <Location size={16} />
+          <span className={styles.value}>{location}</span>
+          <span className={styles.middot}>&middot;</span>
+          <DatePicker dateFormat="d-M-Y" datePickerType="single">
```

Comment:
Why do we have a date picker here? It's not wired to anything, so it's pretty hard to see this as functionally useful.

Reply:
My thinking was that this would be a placeholder, and then further iterations would add functionality to this and use it to filter information on the homepage. The idea was that the first implementation of this would be rendering a static header with little to no functionality but more for information display. This essentially could be re-done into a static date display, but when implementing I was thinking more along the line of adding functionality to an already static date picker in the future.

Reply:
Oh I see, I'll redo this in a component. Thanks.